### PR TITLE
fix: Broken edge picking

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
Edge picking was broken due to a change in the object I was checking. Now it is working.

### CODE REVIEW

**Comment by RobPasMue** [comment]
Looks like some objects are missing when picking

**Comment by AlejandroFernandezLuces** [comment]
I can add them in this PR

File: src/ansys/geometry/core/plotting/plotter_helper.py [hunk 1]
```diff
@@ -229,10 +231,13 @@ def compute_edge_object_map(self) -> Dict[pv.Actor, EdgePlot]:
...
+            geom_obj = object.object
             if (
-                isinstance(object, Body)
-                or isinstance(object, MasterBody)
-                or isinstance(object, Face)
+                isinstance(geom_obj, Body)
+                or isinstance(geom_obj, MasterBody)
+                or isinstance(geom_obj, Face)
+                or isinstance(geom_obj, SketchFace)
+                or isinstance(geom_obj, Sketch)
```
Comment (RobPasMue) [inline src/ansys/geometry/core/plotting/plotter_helper.py:240]: Shouldn't it be SketchEdge?

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: src/ansys/geometry/core/plotting/plotter_helper.py (hunk 1 above)
Comment (AlejandroFernandezLuces) [reply src/ansys/geometry/core/plotting/plotter_helper.py:240]: In this part of the code we need to get the objects that have a list of edges as an attribute, not the edges themselves

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**